packages/trader/src/Modules/Trading/Helpers/contract-type.tsx

```diff
@@ -13,6 +14,11 @@ type TItem = {
     value: string;
 };
 
+export const isMajorPairsSymbol = (checked_symbol: string, active_symbols: ActiveSymbols) =>
```

Comment:
should we do `/FRX/i.test(symbol)` instead of this complicated logic? Since minor pair itself doesn't have multipliers anyways?

Reply:
Because this function might be use not only for `Multipliers`. 
I could create it locally in the component where it's being used and check with reg exp (you are correct, because `Multipliers` are not offered for `Minor Pairs`), but instead of this put this function in the nearest files with helpers. As  there it can be used not only for `Multipliers` contract, I decided to leave check for `Major` (to exclude  `Minor`) pairs as per function's name.
Not sure, if this approach is correct, so I'm open for advices.